### ISSUE TITLE
Multi go2 bringup

### ISSUE THREAD

**Issue by Recepalp10** [anujjain-dev/unitree-go2-ros2#1]
Is it possible to bring up go2 robot multiple times using this repo?
Can we spawn robots at the same time in the simulation environment by changing their namespaces?
And I would be very happy if you could help me on how to do this.

**Comment by anujjain-dev** [comment]
Yes it is possible to spawn multiple go2 robot configurations in the simulation. For this we need to change the namespace, tweak spawning arguments and controllers namespaces.
Thanks for your input. Will surely add this feature soon to the repository.
